macros/contexts/contextLimitedRadical.pl

```diff
@@ -128,6 +128,25 @@ sub _contextLimitedRadical_init {
 	};
 }
 
+###########################
+#
+# Convenience
+#
+# Pass $a,$b, get Formula("$a sqrt($b)") but simplified
+
+sub preprad {
+	my $a = shift;
+	my $b = shift;
+	if ($a == 0 or $b == 0) { return Formula("0"); }
+	my $sign        = ($a > 0)       ? ''    : '-';
+	my $simplifieda = (abs($a) == 1) ? $sign : $a;
+	if ($b == 1) {
+		return Formula("$a");
+	} else {
+		return Formula("$simplifieda sqrt($b)");
+	}
```

Comment:
It would be better to unpack `$a` and `$b` on the first line together.

Also, `$sign` and `$simplifieda` are only used in the `else` clause, and so they should be in its scope only.

I don't really see the need for the `$sign` variable that is used only once.  Just use its value where `$sign` is used that one time.

Finally, instead of an `if ... else ...` with a return statement in each one, this could just do `return ... if ...`.

So this could be condensed to
```suggestion
	my ($a, $b) = @_;
	return Formula("0")  if $a == 0 or $b == 0;
	return Formula("$a") if $b == 1;
	my $simplifieda = abs($a) == 1 ? ($a > 0 ? '' : '-') : $a;
	return Formula("$simplifieda sqrt($b)");
```